client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -51,23 +44,19 @@ interface Props {
 	hostingFlow: boolean;
 }
 
-function getPlanTypes( flowName: string | null, hideFreePlan: boolean, hostingFlow: boolean ) {
+function getPlansIntent( flowName: string | null, hostingFlow: boolean ): PlansIntent | null {
 	switch ( flowName ) {
 		case START_WRITING_FLOW:
 		case DESIGN_FIRST_FLOW:
-			return hideFreePlan
-				? [ TYPE_PERSONAL, TYPE_PREMIUM, TYPE_BUSINESS ]
-				: [ TYPE_FREE, TYPE_PERSONAL, TYPE_PREMIUM, TYPE_BUSINESS ];
+			return 'plans-blog-onboarding';
 		case NEWSLETTER_FLOW:
-			return [ TYPE_FREE, TYPE_PERSONAL, TYPE_PREMIUM ];
+			return 'plans-newsletter';
```

Comment:
Could we create a shareable constant for these slugs?

EX.

export const PLANS_NEWSLETTER = "plans-newsletter";
export const PLANS_LINK_IN_BIO = "plans-link-in-bio";

etc.

Reply:
thanks.. maybe down the line. too early to over-engineer here

Reply:
to be clear - we'll engineer a lot more around this going forward. also include a README in the end to cement it, depending on what stays. "intent" variables will likely end up being exported from plans data-store package